api/router.go

```diff
@@ -97,6 +97,17 @@ func NewRouter(s *Server) *mux.Router {
 	r.Methods("GET").Path("/v1/hosts").Handler(f(schemas, s.NodeList))
 	r.Methods("GET").Path("/v1/hosts/{id}").Handler(f(schemas, s.NodeGet))
 
+	r.Methods("GET").Path("/v1/nodes").Handler(f(schemas, s.MountNodeList))
```

Comment:
The old `/v1/hosts` will be removed later, though we need some UI change to accommodate this. You can use the name `NodeList` and `NodeGet`, just rename the previous one to `HostList`/`HostGet` (in a separate commit). MountNode sounds weird.